### ISSUE TITLE
Cannot find any todos

### ISSUE THREAD

**Issue by AP-Velocity** [fabiospampinato/vscode-projects-plus-todo-plus#6]
Running VSC 1.30.2 on Windows

Running `Projects: Todo` returns
`You don't have any todo across your projects`

Running `Projects: Embedded  Todos` returns
`p.default.embedded.get is not a function`

Tried both the default and these settings:
```JSON
{
    "projects.checkPaths": true,
    "projects.configPath": "C:/Users/*username*/vscode_projects.json",
    "projects.useTilde": false
}
```

In the actual `vscode_projects.json` I have both grouped and single projects, with absolute paths, which are using TODO files in the roots of the paths provided, which Todo+ itself is picking up and Projects+ is able to navigate to the folder.

**Comment by fabiospampinato** [comment]
I can reproduce the issue, but running `Projects: Embedded Todos` doesn't show an error for me 🤔 let's see what the issue might be...

**Comment by fabiospampinato** [comment]
Should be fixed now in v1.6.10.

Thanks for reporting this, I forgot to update this extension after the recent changes I made in its dependencies.
